packages/signals/src/sub.js

```diff
@@ -0,0 +1,134 @@
+// TODO: implement
```

Comment:
This is for future standalone implementation of the `sub$()` function. Right now this code is not used